casper/src/main/scala/coop/rchain/casper/engine/Running.scala

```diff
@@ -163,7 +163,7 @@ object Running {
       if (r.doIgnore) {
         logIgnore(r.status)
       } else {
-        logSuccess >> processHash
+        logSuccess >> processHash.void
```

Comment:
I never use void before. It make sure return `Unit`, right ?

Reply:
Yes, instead of `>> ().pure[F]`